src/c++/perf_analyzer/genai-perf/tests/test_llm_inputs.py

```diff
@@ -261,8 +261,8 @@ def test_random_synthetic(self):
             LlmInputs.DEFAULT_RANDOM_SEED,
         )
 
-        # 785 is the num of tokens returned for the default seed
-        assert synthetic_prompt_tokens == 785
+        # 550 is the num of tokens returned for the default seed
```

Comment:
@nv-braf, I updated this test to use the new number of tokens, since it has been failing since we changed the default standard deviation. If this is not right or you want me to take it out of this PR (I cannot validate the number of tokens expected), let me know.

Reply:
This was updated in a different PR.
Are you update to date with main?

Reply:
That change hasn't been pushed to main yet (waiting until a CI is place). I'm fine having this go here and I'll fix the merge conflict when I push my story.

Reply:
bah. thanks @nv-braf